railties/lib/rails/generators/rails/controller/controller_generator.rb

```diff
@@ -11,7 +15,9 @@ class ControllerGenerator < NamedBase # :nodoc:
       check_class_collision suffix: "Controller"
 
       def create_controller_files
-        template "controller.rb", File.join("app/controllers", class_path, "#{file_name}_controller.rb")
+        destination = File.join("app/controllers", class_path, "#{file_name}_controller.rb")
+        template "controller.rb", destination
+        primary_file(destination)
```

Comment:
This obstructs the flow a bit too much for me.

How about adding a `primary_template` method? Or overriding `template` so we can pass a `primary_file: true` option?

Or perhaps `primary_file` can just return the destination. So we can do `template "controller.rb", primary_file(File.join(…))`

Reply:
Is it possible to open multiple files in the editor too? It might be nice to open the whole shebang after a scaffold generator has run for instance.

Then it could make sense for `template` to collect files to open after the generator has finished.

Reply:
That it depends on the editor whether or not can open multiple files (I'm not sure if it is possible with an editor without tab functions)  
It is difficult to check the operation with various editors, so I would like to avoid supporting it. 

`template "controller.rb", primary_file(File.join(…))` seems nice. I will update.

Reply:
My bad. In order to generate a file, need to run `template` first. So `template "controller.rb", primary_file(File.join(…))` does not work.  
Instead, I added the `primary_template` method.